events/us_arboretum.py

```diff
@@ -0,0 +1,319 @@
+                          'Event Organizers': 'US National Arboretum'}
+            event = get_more_info(event_website, event_data)
+            multi_events.append(event)
+        return multi_events
+    else:
+        event_data = {'Event Name': event_name,
+                      'Event Website': event_website,
+                      'Event Start Time': event_start_time,
+                      'Event End Time': event_end_time,
+                      'Event Start Date': dates,
+                      'Event End Date': dates,
+                      'All Day Event': False,
+                      'Event Venue Name': 'US National Arboretum',
+                      'Timezone': 'America/New_York',
+                      'Event Organizers': 'US National Arboretum'}
+
+        event = get_more_info(event_website, event_data)
+        return event
+
+
+def get_more_info(event_website, event_data):
+    r = get_request(event_website)
+    soup = BeautifulSoup(r.content, 'html.parser')
+    div = soup.find('div', {'class': 'col-sm-9 col-sm-push-3 content'})
+    text_with_time = find_times_text(div)
+    event_description = get_description(div)
+    event_cost = find_event_cost(text_with_time)
+    dates_text = find_dates_text(div)
+    evnt_start_time = event_data['Event Start Time']
+    evnt_end_time = event_data['Event End Time']
+    if event_data['Event Start Date'] == '':
+        try:
+            dates = get_date(dates_text)
+        except Exception:
+            dates = []
+    else:
+        dates = []
+    if evnt_start_time == 0 and evnt_end_time == 0:
+        try:
+            evnt_start_time = get_start_time(text_with_time)
+            evnt_end_time = get_end_time(text_with_time)
+        except Exception:
+            pass
+    if len(dates) > 0:
+        for idx, val in enumerate(dates):
+            multi_events = []
+            multi_event_data = event_data.update({
+                'Event Description': event_description,
+                'Event Start Date': dates[idx],
+                'Event End Date': dates[idx],
+                'Event Start Time': evnt_start_time,
+                'Event End Time': evnt_end_time,
+                'Event Venue Name': 'US National Arboretum',
+                'Event Cost': event_cost,
+                'Event Currency Symbol': "$",
+                'Event Category': " "})
+            multi_events.append(multi_event_data)
+        return multi_events
+    elif len(dates) == 0 and event_data['Event Start Date'] != 0:
+        event_data.update({
+            'Event Description': event_description,
+            'Event Start Time': evnt_start_time,
+            'Event End Time': evnt_end_time,
+            'Event Venue Name': 'US National Arboretum',
+            'Event Cost': event_cost,
+            'Event Currency Symbol': "$",
+            'Event Category': " "})
+        return event_data
+    else:
+        pass
+
+
+def main():
+    try:
+        events = get_events()
```

Comment:
@madhuqmar Looks like `get_events` is missing its argument here so I'm unable to run/test